cinema/views.py

```diff
@@ -34,6 +38,25 @@ class MovieViewSet(viewsets.ModelViewSet):
     queryset = Movie.objects.all()
```

Comment:
The `MovieViewSet.queryset` should be optimized to prevent N+1 query problems when accessing related fields.